### ISSUE TITLE
ansible-galaxy install fails because it can't find galaxy/data/readme

### ISSUE THREAD

**Issue by chrismeyersfsu** [ansible/ansible#11549]
##### Issue Type:

"Bug Report"
##### Ansible Version:

`ansible 2.0.0`
`1163e38d39e583fe13fb171b9e1494f162ab3604`
##### Ansible Configuration:

Nothing of note
http://pastebin.com/1uz0NUEq
##### Environment:

`osx 10.10.4`
##### Summary:

Running `ansible-galaxy install` from devel fails because it can't find galaxy/data/readme
##### Steps To Reproduce:
- checkout ansible
- run make install
- run `ansible-galaxy install`
##### Expected Results:

For the help contents to be printed out
##### Actual Results:

```
[ERROR]: ERROR! Could not open readme: [Errno 2] No such file or directory:
'/usr/local/lib/python2.7/site-packages/ansible-2.0.0-py2.7.egg/ansible/galaxy/data/readme'
```

**Comment by bcoca** [comment]
closing as dupe of #11179, this has not been re-implemented yet in 2.0

**Comment by kisoku** [comment]
@bcoca this is a completely separate issue and still exists in HEAD today.

**Comment by kisoku** [comment]
The issue is that lib/ansible/galaxy/data/readme is not being installed into the egg. setup.py needs to add this file as a data_file

**Comment by bong0** [comment]
@bcoca I can reproduce this issue still in ansible 2.0.0 as shipped by this fedora userbuild: https://copr.fedoraproject.org/coprs/toshio/ansible-upstream/build/111870/

**Comment by noqcks** [comment]
Im still getting this issue today. Updated to ansible v2.0.0 today via pip and getting this issue on multiple machines.

```
ERROR! Could not open readme: [Errno 2] No such file or directory: '/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/ansible/galaxy/data/readme'
```

(note: this happens without virtualenv as well)

**EDIT**: issue fixed by 2.0.1 noted here: https://github.com/ansible/ansible/issues/13419#issuecomment-1709794990
